student.rb

```diff
@@ -1,15 +1,11 @@
-class Student
+require_relative 'people_in_school'
+class Student <  PeopleInSchool
   attr_reader :age, :phase
   attr_accessor :name
 
   def initialize(options = {})
+    super
     @phase = 1
```

Comment:
The code can utilize constants to DRY this source code up a little and actually make it possible to complete remove the initialize code from this subclass:

In each subclass of `PeopleInSchool` (`Student`, `Teacher`) define a constant called `PHASE`, e.g. `PHASE = 3`

In the super class (`PeopleInSchool`) add the following line to the `initialize` method:
`@phase = self.class::PHASE` 

In conjunction with the constant declaration inside the subclass, the super class essentially delegates setting of the `@phase` instance variable to the value of the `PHASE` constant of the class that's getting initialized, in this case a `Teacher` or `Student`. 

`self.class::PHASE` means, "Grab the value of constant in the namespace (`::`) of the class of which I am a member.